modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/form21a_spec.rb

```diff
@@ -24,7 +25,8 @@
         post('/accredited_representative_portal/v0/form21a', params: valid_json, headers:)
 
         expect(response).to have_http_status(:ok)
-        expect(JSON.parse(response.body)).to eq('result' => 'success')
+        expect(parsed_response).to eq('result' => 'success')
+        expect(InProgressForm.exists?(in_progress_form.id)).to be false
```

Comment:
Could you do this instead by checking whether the `/show` route is `nil` for this form? That'd fit the intention of a request spec a bit closer